packages/oauth/src/tokenSession.ts

```diff
@@ -5,7 +5,7 @@
 
 import { parseIdTokenData, decodeIDToken } from "./tokenHelpers"
 
-interface AuthData {
+type AuthData = {
```

Comment:
why you change the interfaces to type?
please have a look here that interface is always over the type.
https://www.totaltypescript.com/type-vs-interface-which-should-you-use#default-to-type-not-interface